TechliftTelegramBot/Controllers/TelegramBotWebhookController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Telegram.Bot;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.ReplyMarkups;
+using Microsoft.Extensions.Configuration;
+using TechliftTelegramBot.Services;
+using Microsoft.Extensions.Logging;
+using TechliftTelegramBot.Models;
+
+
+namespace TechliftTelegramBot.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TelegramBot : ControllerBase
+    {
+        private readonly ILogger _logger;
+        private readonly IConfiguration _config;
```

Comment:
rather than injecting IConfiguration, create a model to store our overall config and inject an object of that using IOptions<>

Reply:
[Check here](https://github.com/techlift-tech/TelegramBotForPokerAnalysis/commit/bda87ee1dfa450689b1b253d59be2b4e0690f348#r50027122)